src/main/java/pt/tqs/g205/resources/PratoResource.java

```diff
@@ -23,13 +24,39 @@
   
   /**
    * Endpoint para obter todos os pratos.
-   * @return JSON com descricao sumaria de todos os pratos.
+   * @return todos os pratos.
    */
   @RequestMapping(value = "", method = RequestMethod.GET)
-  public ResponseEntity<List<Prato>> getAll() {
+  public ResponseEntity<?> getAll() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://sonarcloud.io/organizations/bdomingues-github/rules#rule_key=squid%3AS1452))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=pt.tqs.g205%3Arestapi&pullRequest=6&issues=AWNEkG2fXOBV_Yrh_0ke&open=AWNEkG2fXOBV_Yrh_0ke)

[ISSUE_KEY]: AWNEkG2fXOBV_Yrh_0ke